tests/Test/AbstractIntegrationTestCase.php

```diff
@@ -24,7 +24,6 @@
 use PhpCsFixer\Linter\CachingLinter;
 use PhpCsFixer\Linter\Linter;
 use PhpCsFixer\Linter\LinterInterface;
-use PhpCsFixer\Linter\ProcessLinter;
 use PhpCsFixer\PhpunitConstraintIsIdenticalString\Constraint\IsIdenticalString;
```

Comment:
```suggestion
use PhpCsFixer\Linter\ProcessLinter;
use PhpCsFixer\PhpunitConstraintIsIdenticalString\Constraint\IsIdenticalString;
```

Reply:
I wanted to apply the change myself and merge the PR, but looks like I have no rights to do it